app/src/main/java/com/example/wrk/fragments/FeedFragment.java

```diff
@@ -178,14 +181,13 @@ protected void queryWorkouts() throws ParseException {
         performedQuery.include(WorkoutPerformed.KEY_USER);
         performedQuery.include(WorkoutPerformed.KEY_WORKOUT);
         // to remove posts that have already been added
-        performedQuery.whereNotContainedIn(WorkoutPerformed.KEY_USER, recentFollowedWorkouts);
-        performedQuery.whereNotContainedIn(WorkoutPerformed.KEY_WORKOUT, mostPopularWorkouts);
+        performedQuery.whereNotContainedIn(WorkoutPerformed.KEY_USER, recentFollowedUsers);
+        performedQuery.whereNotContainedIn(WorkoutPerformed.KEY_WORKOUT, popularTemplates);
```

Comment:
BUG FIX: Incorrect filtering use within queryWorkouts. Line 181 is searching individually for if a user is not inside recentFollowedWorkouts. Line 182 is searching for if a particular template is not inside mostPopularWorkouts.

Issue with Line 181: recentFollowedWorkouts is a List of WorkoutPerformed, but is not a list of ParseUsers which it would need
FIX: create a new list that holds the users for each WorkoutPerformed in the recentFollowedWorkouts list. Changing the list to the new one, recentFollowedUsers allows it to properly search for users as it should to ensure there are no duplicates

Issue with Line 182: mostPopularWorkouts is a List of WorkoutPerformed, not a list of WorkoutTemplate. In order to properly search whether or not the template exists already, it needs to have a list of templates instead.
FIX:  I had a list of popularTemplates that was used when querying for the most popular workout posts, so I just used that same List since it would already be populated by the time the queryWorkouts() function was called.

Summary: Found issue with filtering to remove any posts that had already been found. Needed to change the current filter lists to ones that were a list of ParseUsers or WorkoutTemplates to properly fit is as it should.